src/store/modules/user.ts

```diff
@@ -0,0 +1,75 @@
+import { ActionContext, Module } from 'vuex'
+import { User } from './types'
+
+export enum RoleId {
```

Comment:
you already have the enum here, can we remove the RoleId type then?

Reply:
What enum, I don't understand?